DOCS-983,DOCS-911: Fix Color Detection tut (#1668)
viamrobotics/docs: docs/tutorials/services/try-viam-color-detection.md

Be aware that if you are running out of time during your rental, you can [extend your rover rental](/try-viam/reserve-a-rover/#extend-your-reservation) as long as there are no other reservations.
{{< /alert >}}

## Enable the cameras

> Before configuring color detection, enable the rover's camera to get a better sense of what it perceives as it performs color classification.

Reviewer: It's not classification 
```suggestion
Before configuring color detection, enable the rover's camera to get a better sense of what it perceives.
```

> Before configuring color detection, we recommend that you enable the rover's camera to get a better sense of what it perceives as it performs color classification.

Reviewer 2: ```suggestion
Before configuring color detection, enable the rover's camera to get a better sense of what it perceives as it performs color classification.
```
[opt]

* If you are running this tutorial with a [rented Viam Rover](https://app.viam.com/try), enable both provided cameras: the front-facing camera and the overhead cam.
  In the `viam_base` component panel under the **Control** tab, enable both the `cam` for the front-facing camera and the `overhead-cam:overheadcam` for an overhead view of your rover.

>   ![Screenshot of the viam_base component panel showing both the 'cam' and 'overheadcam' camera feeds enabled.](try-viam/try-viam/enable-both-cameras.png)

Reviewer: Alt texts should not beging with photo of or screenshot of, because then screenreaders will say photo of screenshot of
```suggestion
  ![The viam_base component panel showing both the 'cam' and 'overheadcam' camera feeds enabled.](try-viam/try-viam/enable-both-cameras.png)
```

Author: Also expanded this to 12 other instances across these two pages.

>   You can also view and control the camera streams from the [individual camera components](/try-viam/try-viam-tutorial/#camera-control).

Reviewer 2: ```suggestion
  You can also view and control the camera streams from the [individual camera component panels](/try-viam/try-viam-tutorial/#camera-control).
```

* If you are running this tutorial on [your own Viam Rover](/try-viam/rover-resources/), enable the front facing camera.

>   If you are using the `ViamRover` [fragment](https://app.viam.com/fragments) with your rover, the front facing camera is named `cam` and can be enabled in the `viam_base` component panel under the **Control** tab.

Reviewer 2: ```suggestion
  If you are using the `ViamRover` [fragment](/try-viam/rover-resources/rover-tutorial-fragments/) with your rover, the front facing camera is named `cam` and can be enabled in the `viam_base` component panel under the **Control** tab.
```

## Add the vision service to detect a color

This tutorial uses the hex color `#7a4f5c` (a reddish color).

> **Hex color #7a4f5c**: {{<imgproc src="/tutorials/try-viam-color-detection/7a4f5c.png" resize="300x" declaredimensions=true alt="A color swatch for the color that you will be detecting with your color detector. It's a reddish, maroon color.">}}

Reviewer 2: ```suggestion
**Hex color #7a4f5c**: {{<imgproc src="/tutorials/try-viam-color-detection/7a4f5c.png" resize="200x" declaredimensions=true alt="A color swatch for the color that you will be detecting with your color detector. It's a reddish, maroon color.">}}
```
[opt out of diff nit] I just think this box looks oddly big, almost like a big redaction

Scroll to the **Create Service** section.
To create a [vision service](/services/vision/):

1. Select `Vision` as the **Type**.
1. Enter `my_color_detector` as the **Name**.
1. Select **Color Detector** as the **Model**.
1. Click **Create Service**.
1. In the resulting vision service panel, click the color picker box to set the color to be detected.
   For this tutorial, set the color to `rgb(122, 79, 92)` or use hex code `#7a4f5c`.

1. Then, set **Hue Tolerance** to `0.06` and **Segment Size px** to `100`.

Your configuration should look like the following:

> {{<imgproc src="/try-viam/try-viam/vision-service-config.png" resize="x500" alt="Screenshot of the vision service configuration showing the color set to a reddish color, the hue tolerance set to 0.06, and the segment size set to 100.">}}

Reviewer: Please retake this screenshot and resize the window to a smaller width before you take it

Author: Thanks for this, you're right -- it's tough to read on mobile! Switched to a smaller width screenshot!

Scroll to the **Create Service** section.
To create a [vision service](/services/vision/):

1. Select `Vision` as the **Type**.
1. Enter `my_color_detector` as the **Name**.
1. Select **Color Detector** as the **Model**.
1. Click **Create Service**.
1. In the resulting vision service panel, click the color picker box to set the color to be detected.
   For this tutorial, enter the following values:

   Color | Value
   ----- | -----
   R | 122
   G | 79
   B | 92

>    Or use the contextual menu in the lower-right of the pop-up window to switch to hex mode, and enter the hex code `#7a4f5c`.

Reviewer 2: This is too specific since the UI varies depending on browser and OS. For me, hex code goes in the top :D 
```suggestion
   Or enter the equivalent hex code: `#7a4f5c`
```

Author: Ah sorry, you put this right in the ticket!! 🤦  Fixed, thank you!
